requirements-ci.txt

```diff
@@ -8,7 +8,7 @@ autograd
 toml
 appdirs
 semantic_version
-autoray
+autoray==0.6.8
```

Comment:
I'm aware this is a recent bug introduced by autoray, but is there a separate PR addressing this CI change? Might be better to merge master into this branch to fix the CI issues instead.

Reply:
This is just a workaround to let CI work until that PR goes in!